pkg/kube/cluster-init.sh

```diff
@@ -466,6 +484,8 @@ else
             else
                 logmsg "k3s is down and restart count exceeded."
             fi
+        else
+          check_overwrite_nsmounter
```

Comment:
i don't think this call should be here.
it should be in 2 places, first on is inside the if [ ! -f /var/lib/longhorn_initialized ]; then ... fi block for the first time the longhorn is downloaded and started; the 2nd place should be inside the if [ $? -eq 1 ]; then ... fi block, and maybe before the "# setup debug ....", so when the device restarts, we also run this, but only once. Unless I misunderstood the function here.

Reply:
In this spot it runs after longhorn is installed and on restarts